backend/src/api/helpers.ts

```diff
@@ -180,8 +181,9 @@ export const sendRegionalAdminNotificationEmail = async (
   });
 
   const fs = require('fs').promises;
+  const client = new S3Client();
   const html = await fs.readFile(
-    '/app/src/email_templates/crossfeed_regional_admin_notification.html',
+    await client.getEmailAsset('crossfeed_regional_admin_notification.html'),
```

Comment:
Are we gonna need the fs.readFile() here if we are grabbing from the S3 bucket?

Reply:
We shouldn't need that.